### PR TITLE
Make nav logo link to home

### PR DESCRIPTION
**Pull request by johnkuney**

### CODE REVIEW

File: v2/src/layout/header.js [hunk 1]
```diff
@@ -152,11 +152,17 @@ const Header = () => {
...
+                data-sal="slide-down"
+                data-sal-delay={200}
+                data-sal-duration="1000"
+                data-sal-easing="ease"
+                className={headerStyles.logo}
+                to="/"
+              >
+                <img src={logo} alt="bitcoincashlogo" />
+              </Link>        
+            <div className={headerStyles.divider}></div>
```
Comment (jasonbcox) [inline v2/src/layout/header.js:165]: For empty tags, just terminate the first tag at the end like this:
`<div className={headerStyles.divider} />`

File: v2/src/layout/header.js [hunk 2]
```diff
@@ -152,11 +152,17 @@ const Header = () => {
...
-            <img
-              src={logo}
-              className={headerStyles.logo}
-              alt="bitcoincashlogo"
-            />
+              <Link
+                data-sal="slide-down"
+                data-sal-delay={200}
+                data-sal-duration="1000"
+                data-sal-easing="ease"
```
Comment (jasonbcox) [inline v2/src/layout/header.js:159]: This confuses me a little. Is this an animation on the link? should this be one container level up from this?

File: v2/src/layout/header.js (hunk 1 above)
Comment (johnkuney) [reply v2/src/layout/header.js]: ah for sure. fixed

File: v2/src/layout/header.js (hunk 2 above)
Comment (johnkuney) [reply v2/src/layout/header.js]: Yeah could point I've moved it up one level so the price widget animates with it

**Pull request merged**